### DISCUSSION TITLE
透过alist的下载较慢，应该如何优化？Through the alist, the download speed is slower. How can it be optimized?

### DISCUSSION THREAD

**Discussion by c469591** [AlistGo/alist#5052] [Q&A]
大家好，我遇到一个问题，
目前我使用rclone将one drive挂载到本地，
然后使用alist将挂载的one drive目录作为alist的一个存储，
透过alist下载文件时，速度无法跑满服务器的宽待，
如果我将文件放在透过nginx架设的网站目录中，也就是放在服务器本地，
之后直接使用ip/文建铭来下载时，速度却可以跑满服务器宽待，
我也试过用alist挂载百度网盘，且我有摆渡付费VIP，速度也是不如我直接从nginx的网站中下载
我不知道应该如何设置来优化从alist的下载速度，请高手指点，感谢！
下面是英文版的说明
Hello everyone, I have encountered a problem. Currently, I am using rclone to mount OneDrive locally. Then, I use alist to store the mounted OneDrive directory as one of alist's storage locations. When downloading files through alist, the speed cannot reach the maximum bandwidth of the server. However, if I place the files in the website directory hosted by nginx on the server itself and directly download them using "ip/filename", then the speed can fully utilize the server's bandwidth.

I have also tried mounting Baidu Netdisk with alist, and I have a paid VIP subscription for Baidu Netdisk. However, even with that setup, the speed is still slower compared to directly downloading from nginx's website.

I don't know how to optimize download speeds from alist. Could someone knowledgeable please provide guidance? Thank you!

**Comment by AsColdnet** [comment]
我也遇到这个情况，不知道咋解决
